client/sites/staging-site/hooks/use-get-lock-query.ts

```diff
@@ -4,7 +4,10 @@ import wp from 'calypso/lib/wp';
 export const USE_STAGING_SITE_LOCK_QUERY_KEY = 'staging-site-lock';
 export const useGetLockQuery = (
 	siteId: number,
-	options: UseQueryOptions< boolean | null, unknown, boolean, ( string | number )[] >
+	options?: Omit<
+		UseQueryOptions< boolean | null, unknown, boolean, ( string | number )[] >,
+		'queryKey' | 'queryFn'
+	>
```

Comment:
If I am understanding correctly, this change is improvement not directly related to other changes in the PR, right?

(just checking in case I missed something)